syntax = "proto3";
package protoData;

option go_package = "golang-client/message";
message Action{
  string action_description = 1;
  int32 duration = 2;
  uint64 start_time = 3;
  uint64 end_time = 4;
}
//If Its plural, repeated proto message needs to be added
message Actions{
  repeated Action actions =1;
}
message EmojiData{
  string emoji_description = 1;
  string emoji_unicode = 2;
}
message ParsedAction{
  EmojiData EmojiList = 1;
}
